StackProgram.cs

```diff
@@ -1,12 +1,38 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Collections;
+using Stack.Interfaces;
+
 
 namespace Stack
 {
     class StackProgram
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            ArrayStack<int> s = new ArrayStack<int>();
+            Action<object> action = new Action<object>(PrintStack);
+            s.Push(5);
+            s.Push(6);
+            s.Push(7);
+            s.Push(8);
+            int sum = 0;
+            s.PrintStack(action);
```

Comment:
make it like this please  s.PrintStack(new Action<object>(PrintStack)); and remove line 18